src/crimson/os/seastore/journal/circular_bounded_journal.h

```diff
@@ -237,14 +186,16 @@ class CircularBoundedJournal : public Journal {
     auto rbm_tail = get_rbm_addr(get_dirty_tail());
     return rbm_written_to >= rbm_tail ?
       rbm_written_to - rbm_tail :
-      rbm_written_to + header.size + get_block_size()
+      rbm_written_to + get_total_size() + get_block_size()
```

Comment:
If that is correct, `get_total_size()` should be `device->get_journal_size() - 2 * get_block_size()`.

The names are confusing, probably:
* get_start_addr -> get_records_start
* get_total_size -> get_records_total_size
* get_available_size -> get_records_available_size
* get_used_size -> get_records_used_size

And group the related interfaces together.

Reply:
> And `get_records_total_size = records_area_size - block_size` because a block is reserved in the records_area to avoid journal_tail and written_to to overlap each other.
>
> If that is correct, get_records_total_size() should be `device->get_journal_size() - 2 * get_block_size()`.

@myoungwon seems not fully addressed^